src/XIVLauncher.Core/Components/SettingsPage/Tabs/SettingsTabSteamTool.cs

```diff
@@ -0,0 +1,141 @@
+    
+    private bool steamFlatpakInstalled = SteamCompatibilityTool.IsSteamFlatpakInstalled;
+    
+    private bool steamFlatpakToolInstalled = SteamCompatibilityTool.IsSteamFlatpakToolInstalled;
+
+    public SettingsTabSteamTool()
+    {
+        Entries = new SettingsEntry[]
+        {
+            steamPath = new SettingsEntry<string>("Steam Path (native)", "Path to the native steam install. Only change this if you have Steam installed in a non-default location.",
+                () => Program.Config.SteamPath ?? Path.Combine(CoreEnvironmentSettings.HOME, ".local", "share", "Steam"), s => Program.Config.SteamPath = s),
+            steamFlatpakPath = new SettingsEntry<string>("Steam Path (flatpak)", "Path to the flatpak Steam installation. Only change this if you have your flatpak Steam installed to a non-default location.",
+                () => Program.Config.SteamFlatpakPath ?? Path.Combine(CoreEnvironmentSettings.HOME, ".var", "app", "com.valvesoftware.Steam", "data", "Steam" ), s => Program.Config.SteamFlatpakPath = s)
+            {    
+                CheckVisibility = () => Program.IsSteamDeckHardware != true && SteamCompatibilityTool.IsSteamFlatpakInstalled,
+            },
+        };
+    }
+
+    public override SettingsEntry[] Entries { get; }
+
+    public override bool IsUnixExclusive => true;
+
+    public override string Title => "Steam Tool";
+
+    public override void Draw()
+    {
+        if (CoreEnvironmentSettings.IsSteamCompatTool)
+        {
+            ImGui.Dummy(new Vector2(10));
+            ImGui.Text("You are currently running XIVLauncher.Core as a Steam compatibility tool.");
+            ImGui.Dummy(new Vector2(10));
+            ImGui.Text("If you are trying to upgrade, you must first update your local install of XIVLauncher.Core. Then launch the local" +
+                        "\nversion, navigate back to this tab, and re-install as a Steam compatibility tool.");
+            ImGui.Text("\nIf you are trying to uninstall, you should likewise launch the native version of XIVLauncher, and click the appropriate" +
+                        "\nuninstall button.");
+            return;
+        }
+        ImGui.Text("\nUse this tab to install XIVLauncher.Core as a Steam compatibility tool.");
+        ImGui.Text("\nAfter you have installed XIVLauncher as a Steam compatibility tool please close XIVLauncher and launch or restart Steam. Find 'Final Fantasy XIV Online'");
+        ImGui.Text("in your steam library and open the 'Properties' menu and navigate to the 'Compatibility' tab. Enable 'Force the use of a specific Steam Play compatibility tool'");
+        ImGui.Text("and from the dropdown menu select 'XIVLauncher.Core'. If this option does not show up then restart Steam and try again. After finishing these steps,");
+        ImGui.Text("XIVLauncher will now be used when launching FINAL FANTASY XIV from steam.");
+        if (Program.IsSteamDeckHardware != true && steamFlatpakInstalled)
```

Comment:
`steamFlatpakInstalled` check probably should be removed there, because when if user did not provide permission to access steam's flatpak this check will be false, so user wouldn't see a guide on how to do it :)

Reply:
Also flatpak can resolve `~` to a homedir in a paths, so `{CoreEnvironmentSettings.HOME}` can also be replaced with that, I think?

Reply:
I'll double check the first thing you mentioned. I think we can still see into that directory an read the file structure, but I admit I might not have checked properly.

As for using ~, flatpak may understand it but dotnet doesn't.

Reply:
Alright, updated it to show the flatpak steam installation instructions as long as we're not looking at a steam deck install.

Running the official flatpak or native will now automatically update the steam tool if it's installed and needs updating.

Also improved version handling so it can tell if the steam tool is official or rb-patched, so they don't automatically update over each other. In other works, RB-patched installs won't automatically update and overwrite official installs, and vice-versa.